style.css

```diff
@@ -391,9 +391,11 @@ input[type=text] {
   width: 50%;
   height: 4.5rem;
   margin: 1.5% 0;
+  background-color: #fff;
 }
 
```

Comment:
- [ ] Kindly make sure to include the live demo link in your README.md file, as per requirement.